### ISSUE TITLE
Can we use better source data?

### ISSUE THREAD

**Issue by chmac** [TripSit/comboapp#7]
There's a few different source of data. I started out with just the resulting combo data, but it's not complete. Apparently the combo app is not so actively being worked on (according to [this issue](https://github.com/TripSit/factsheets/issues/5)). Maybe it's possible to build the data during app build somehow. I'm gathering thoughts here in case anyone else wants to work on this, or so I can come back to it later when I'm feeling inspired!

This could provide some inspiration on where to get data:
https://github.com/TripSit/factsheets/blob/master/routes/index.js

**Comment by Spacebjorn** [comment]
The issue says combogen not comboapp. These are two different things. Combogen well, generates the combochart as html file. The combochart was designed to chose from from common drugs to be easily used at festivals and what not. The intention with the combo charts was never to include a host of drugs on it as it is meant for print. Feel free to create a issue on the combogen page

**Comment by chmac** [comment]
@Spacebjorn Yeah, I know they're two different things, and I meant this for this repo as I originally wrote this app. I realise that there's more data than is used in the combogen app and I'm thinking it would make sense to include that here.

**Comment by chmac** [comment]
@Spacebjorn Can you reopen this issue? I can't do that it seems.
